### ISSUE TITLE
Badges are not accessible

### ISSUE THREAD

**Issue by josebolos** [pa11y/org#8]
If I'm not mistaken the shields.io badges that we use in all our repos are not accessible. I only went as far as testing the green ones - they have a contrast ratio of 2.21:1 I think. 

(There may be other problems with them. I always thought that the font size is too small too, and inconsistent with the rest of the _readme_.)

An example from [https://github.com/pa11y/dashboard](https://github.com/pa11y/dashboard):

![screen shot 2016-09-27 at 11 10 03](https://cloud.githubusercontent.com/assets/1792637/18869254/6a8f1bf8-84a3-11e6-8f88-75bf4ed2cda0.png)
